src/Spryker/Zed/AppMerchant/Persistence/Propel/Schema/spy_app_payment.schema.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<database xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed" xsi:noNamespaceSchemaLocation="http://static.spryker.com/schema-01.xsd" namespace="Orm\Zed\AppPayment\Persistence" package="src.Orm.Zed.AppPayment.Persistence">
+
+    <table name="spy_payment_transfer" idMethod="native" class="SpyPayment">
+        <column name="merchant_reference" type="CHAR" size="36" required="false" description="Has to be false for order items that do not have a merchant"/>
```

Comment:
>Has to be false for order items that do not have a merchant
"false" for the CHAR field? maybe empty/null?

Reply:
The text is misleading, true. Updated